### PR TITLE
My Feet are Cold

### PR DESCRIPTION
**Pull request by KangTut**
# What this PR does
Gives Rev antags socks and winter boots

# Why it should be added to the game
Revs have no speed bonuses compared to syndies or the player mercs. This buff will equip rev players with socks and winter boots for a total of 20% extra speed (the same if a syndie buys a mk. 3 mauler suit).

### CODE REVIEW

File: code/_core/datum/damagetype/ranged/laser/freezegun.dm [hunk 1]
```diff
@@ -4,11 +4,11 @@
 	//The base attack damage of the weapon. It's a flat value, unaffected by any skills or attributes.
 	attack_damage_base = list(
 		LASER = DAMAGE_AXE*0.25,
-		COLD = DAMAGE_AXE*0.75,
+		COLD = DAMAGE_CLUB*1,
```
Comment (BurgerLUA) [inline code/_core/datum/damagetype/ranged/laser/freezegun.dm:7]: Why are there undocumented changes AAAAAAAAAAAAA please remove these freezegun changes.

File: code/_core/datum/loadout/rev_new.dm [hunk 2]
```diff
@@ -158,8 +158,10 @@
 		/obj/item/clothing/head/hat/skimask/black,
 		/obj/item/clothing/hands/gloves/colored/padded/black,
 		/obj/item/clothing/hands/gloves/colored/padded/black/left,
-		/obj/item/clothing/feet/shoes/black_boots,
-		/obj/item/clothing/feet/shoes/black_boots/left,
+		/obj/item/clothing/feet/socks/knee,
+		/obj/item/clothing/feet/socks/knee,
+		/obj/item/clothing/feet/shoes/winter,
```
Comment (BurgerLUA) [inline code/_core/datum/loadout/rev_new.dm:163]: I'm fine with socks but the boots should be jackboots to match the color scheme.

File: code/_core/obj/item/weapon/ranged/laser/freezegun.dm [hunk 3]
```diff
@@ -30,6 +31,39 @@
 
 	value = 1500
 
+
+	attachment_whitelist = list(
```
Comment (BurgerLUA) [inline code/_core/obj/item/weapon/ranged/laser/freezegun.dm:35]: Undocumented changes aaaaaaaaaaaaaaaaaaaaaaaaaaaa please remove these and put them in another PR

**Review by BurgerLUA — CHANGES_REQUESTED** [review]

File: code/_core/obj/item/weapon/ranged/laser/freezegun.dm (hunk 3 above)
Comment (KangTut) [reply code/_core/obj/item/weapon/ranged/laser/freezegun.dm:35]: wait these were already added though. Fuck sorry I'm still new to github.
